config/manager/manager.yaml

```diff
@@ -49,7 +49,7 @@ spec:
             - "--logtostderr"
             - "--exempt-namespace=gatekeeper-system"
             - "--operation=webhook"
-          image: openpolicyagent/gatekeeper:v3.2.0
+          image: openpolicyagent/gatekeeper:v3.3.0-beta.1
```

Comment:
I don't see the updates in deploy/gatekeeper.yaml

Reply:
thanks! fixed